tests/test_check_cloudformation.py

```diff
@@ -0,0 +1,52 @@
+import os
+import shutil
+
+from click.testing import CliRunner
+
+from commands.check_cloudformation import check_cloudformation as check_cloudformation_command
+from tests.conftest import BASE_DIR
+
+
+def test_runs_all_checks_when_given_no_arguments():
+    result = CliRunner().invoke(check_cloudformation_command)
+
+    assert ">>> Running all checks" in result.output
+    assert ">>> Running lint check" in result.output
+
+
+def test_prepares_cloudformation_templates():
+    def path_exists(path):
```

Comment:
Nitpicking, but if you use the new python 3 pathlib stuff, this is superfluous

```
from pathlib importh Path

path = Path("my/path")

if path.exists():
 ...

Reply:
Done. Code more readable now :-)